rest/data/project.go

```diff
@@ -169,6 +129,51 @@ func CreateProject(ctx context.Context, env evergreen.Environment, projectRef *m
 	return nil
 }
 
+func tryCopyingContainerSecrets(ctx context.Context, settings *evergreen.Settings, existingSecrets []model.ContainerSecret, pRef *model.ProjectRef) error {
+	// TODO (PM-2950): remove this temporary error-checking once the AWS
+	// infrastructure is productionized and AWS admin settings are set.
+	smClient, err := cloud.MakeSecretsManagerClient(settings)
+	if err != nil {
+		return gimlet.ErrorResponse{
+			StatusCode: http.StatusInternalServerError,
+			Message:    errors.Wrap(err, "setting up Secrets Manager client to store newly-created project's container secrets").Error(),
+		}
+	}
+	defer smClient.Close(ctx)
+
+	vault, err := cloud.MakeSecretsManagerVault(smClient)
```

Comment:
Previously we only did this if smClient wasn't nil; do we still need to do that?

Reply:
`cloud.MakeSecretsManagerClient` will return either an error or a non-nil object, but not both, so we don't need to check for nil here.